README.md

```diff
@@ -7,31 +7,95 @@
 [![CI](https://github.com/code-pushup/cli/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/code-pushup/cli/actions/workflows/ci.yml?query=branch%3Amain)
 [![Codecov](https://codecov.io/gh/code-pushup/cli/branch/main/graph/badge.svg?token=Y7V489JZ4A)](https://codecov.io/gh/code-pushup/cli)
 
-🔎🔬 **Quality metrics for your software project.** 📉🔍
+<h1 align="center">🔎🔬Code quality integrations for any tool📉🔍</h1>
```

Comment:
IMHO this huge centered heading looks terrible :-1: Paired with the overall excess of large and/or bold text and repeated phrases, this whole section of the README now looks like a crummy ad :grimacing: 

![image](https://github.com/code-pushup/cli/assets/34691111/ce645f65-45b7-4fd1-b0e8-3228357f5167)

Reply:
moved it as bold to the top